cheddar/cheddar-server/src/test/java/com/clicktravel/cheddar/server/http/filter/features/ContainerFeatureSetRequestFilterTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2014 Click Travel Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.clicktravel.cheddar.server.http.filter.features;
+
+import static com.clicktravel.common.random.Randoms.randomId;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.verifyStatic;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.core.MultivaluedHashMap;
+import javax.ws.rs.core.MultivaluedMap;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import com.clicktravel.cheddar.request.context.features.FeatureSetContext;
+import com.clicktravel.cheddar.request.context.features.FeatureSetContextHolder;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ FeatureSetContextHolder.class })
+public class ContainerFeatureSetRequestFilterTest {
+
+    private static final String TRAVEL_CLOUD_FEATURE_SET_ID_HEADER = "Travel-Cloud-Feature-Set-Id";
```

Comment:
This identifier contains a marketing term (which is unstable) and is also specific to development at Click Travel and not applicable to a general audience. This identifier should instead be abstract, conveying its role. Perhaps `Active-Feature-Set` or `Enabled-Feature-Set`